ast/format_test.go

```diff
@@ -0,0 +1,130 @@
+package ast_test
+
+import (
+	"testing"
+
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/parser"
+	"github.com/pkg/errors"
+)
+
+var skip = map[string]string{
+	"array_expr":        "without pars -> bad syntax, with pars formatting removes them",
+	"conditional":       "how is a conditional expression defined in spec?",
+	"duration_multiple": "how are multiple duration values meant to be used?",
+}
+
+func TestFormat(t *testing.T) {
+	testCases := []struct {
+		name  string
+		query string
```

Comment:
These test cases look great!

Small nit, can you rename `query` to `script` since I want the test writer thinking that these could be any Flux scripts, not just queries?